backend/src/logUtils.ts

```diff
@@ -5,13 +5,37 @@ import { YeomanUI } from "./yeomanui";
 
 
 module.exports = ( output: Output, yeomanUi: YeomanUI) => {
-	function getMessage(args: any) {
+
+	const defaultColors = {
+		skip: 'yellow',
+		force: 'yellow',
+		create: 'green',
+		invoke: 'bold',
+		conflict: 'red',
+		identical: 'cyan',
+		info: 'gray'
+	  };
+
+	function pad(status: string) {
+		const max = 'identical'.length;
+		const delta = max - status.length;
+		return delta ? ' '.repeat(delta) + status : status;
+	}
+	  
+	function getPrefix(status: string) {
+		const color = _.get(defaultColors, status);
+		if (_.isEmpty(color)) return '';
+		return pad(status) + ' ';
```

Comment:
`${pad(status)} `

where is pad defined ?

Reply:
up --> line 19